content/blog/2022-02-09-predictkube-scaler.md

```diff
@@ -0,0 +1,163 @@
++++
+title = "Introducing PredictKube - an AI-based predictive autoscaler for KEDA made by Dysnix"
+date = 2022-02-09
+author = "Daniel Yavorovych (Dysnix), Yuriy Khoma (Dysnix), Zbynek Roubalik (KEDA), Tom Kerkhove (KEDA)"
+aliases = [
+"/blog/introducing-predictkube-an-ai-based-predictive-autoscaler-for-keda-made-by-dysnix"
+]
++++
+
+## Introducing PredictKube—an AI-based predictive autoscaler for KEDA made by Dysnix
+
+[Dysnix](https://dysnix.com/) has been working with big high-loaded projects for a long time,
+and the efficient scaling demand is what their team comes across each day.
+The engineers have understood that dealing with traffic fluctuations and preparations of infrastructure by bare hand is
+totally inefficient as you need to deploy more resources _before_ the traffic increases,
+not at the moment the event happens. It is too late to scale when the big traffic wave has already come.
+
+So, Dysnix started thinking about predictive autoscaling. To scale in time, you have to scale beforehand—or else,
+you'll have to deal with another problem that's familiar to each cloud inhabitant.
+The overprovision of resources is used to cover any traffic changes and results in bills and thrills connected with unexpected expenses.
+Can this process be more controllable?
+
+And when it comes to deciding how to wrap up this solution, Dysnix decided to rely on KEDA as it is the most
+universal and applicable component for application autoscaling in Kubernetes.
+
+KEDA is being used as a component on the client side of PredictKube that is responsible for transferring requests
+and scaling replicas.
```

Comment:
I think that these 3 paragraphs aren't needed if you mention overprovisioning above (in my previous comment).

Reply:
Thanks for your suggestion, Aaron, but we'd like to leave a part about the KEDA role for PredictKube.